vector/src/main/res/values/strings.xml

```diff
@@ -879,37 +879,37 @@
     <string name="room_add_matrix_apps">Add Matrix apps</string>
     <string name="settings_labs_native_camera">Use native camera</string>
 
-    // share keys
+    <!-- share keys -->
     <string name="you_added_a_new_device">You added a new device \'%s\', which is requesting encryption keys.</string>
     <string name="your_unverified_device_requesting">Your unverified device  \'%s\' is requesting encryption keys.</string>
     <string name="start_verification">Start verification</string>
     <string name="share_without_verifying">Share without verifying</string>
     <string name="ignore_request">Ignore request</string>
 
-    // conference call
+    <!-- conference call -->
     <string name="conference_call_warning_title">Warning!</string>
     <string name="conference_call_warning_message">Conference calling is in development and may not be reliable.</string>
 
-    // slash commands
+    <!-- slash commands -->
     <string name="command_error">Command error</string>
     <string name="unrecognized_command">Unrecognized command: %s</string>
 
-    // notification statuses
+    <!-- notification statuses -->
     <string name="notification_off">Off</string>
     <string name="notification_on">On</string>
     <string name="notification_noisy">Noisy</string>
 
     <string name="encrypted_message">Encrypted message</string>
 
-    // groups creation
+    <!-- groups creation -->
     <string name="create">Create</string>
     <string name="create_community">Create Community</string>
     <string name="community_name">Community name</string>
     <string name="community_name_hint">Example</string>
     <string name="community_id">Community Id</string>
     <string name="community_id_hint">example</string>
 
-    // group details
+    <!-- group details -->
     <string name="group_details_home">Home</string>
     <string name="group_details_people">People</string>
     <string name="group_details_rooms">Rooms</string>
```

Comment:
Think of adding the strings in Weblate and prevent the community in the room Riot Translation, if it is not already done.

Reply:
yes, thanks